package.json

```diff
@@ -9,10 +9,11 @@
     "npm": ">=1.3.0 <3.0.0"
   },
   "devDependencies": {
-    "eslint": "^7.32.0",
-    "grunt": "^1.0.1",
-    "grunt-cli": "^1.2.0",
-    "grunt-contrib-qunit": "^4.0.0",
-    "qunit": "^2.9.3"
+    "eslint": "^8.3.0",
+    "puppeteer": "^13.5.1",
```

Comment:
Is it intentional for Puppeteer to be there? I can’t see it being used anywhere in the project.

Reply:
`puppeteer` is used by `grunt-contrib-qunit`. We wanted to pin a more recent version than required by `grunt-contrib-qunit`.

Reply:
Thank you, that’s very interesting for potential automated accessibility tests as there are many more options with Puppeteer than Selenium.

Reply:
Also, the default version installed with `grunt-contrib-qunit` raises a warning:
```
npm WARN deprecated puppeteer@9.1.1: Version no longer supported. Upgrade to @latest
```